Cargo.toml

```diff
@@ -14,13 +14,13 @@ documentation = "https://docs.rs/crate/parsec-tool"
 [dependencies]
 ansi_term = "0.12.1"
 atty = "0.2.14"
-clap = "3.0.0-beta.1"
-parsec-client = "0.14.0"
+clap = "2.34.0"
```

Comment:
I'm assuming we're not interesting in fixing the issues highlighted by `cargo-audit` in the nightly run? I don't think it's a critical thing to fix, just wanted to make sure we're doing this consciously

Reply:
Here is what I thought, fixing the nightly would be on parsec service, and it will require moving `clap` from this version or not using `structopt` at all. And `parsec-tool` should always align with parsec crates.
So for `parsec-tool` It shouldn't be the place to fix that. ( it will always follow the service)
so the question is Do we want to fix that audit issue for this release? if yes, let's hold this PR until we fix it in the service.

Reply:
Ok, that makes sense - let's just go with the current choice of libraries, we can fix for 1.2.0 and ignore the `cargo-audit` stuff until then.

What about the yanked version of `cpufeatures`? Is that also common with the service?

Reply:
The used `cpufeatures` version `0.2.5` is not yanked (so far 🙄), and this crate isn't used in the service. but it's used in the `e2e_tests` which it's `lockfile` is ignored

Reply:
Excellent, ok, thank you!

Reply:
You're welcome 😄!